federation/commons/src/main/java/org/codice/ddf/admin/commons/sources/SourceHandlerCommons.java

```diff
@@ -38,6 +38,8 @@
 
 public class SourceHandlerCommons {
 
+    private static final RequestUtils REQUEST_UTILS = new RequestUtils();
```

Comment:
I agree this looked strange to me as well. It's being used to generate a static map, which is being used in a static method that's being used to generate descriptions for each method (probe, test, persist) for each source.
I could change this (I'm not decided either way) but it would be a rather involved change - instantiation SourceHandlerCommons in a lot of places that are currently using a static method reference.